packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -388,6 +389,43 @@ class RelativePositionedTransition extends AnimatedWidget {
   }
 }
 
+/// Animated version of a [DecoratedBox] that animates the different properties
+/// of its [Decoration].
+class DecoratedBoxTransition extends AnimatedWidget {
+  /// Creates an animated [DecorationBox] whose [Decoration] animation updates
+  /// the widget. 
+  /// 
+  /// `decoration` and `position` cannot be null.
+  /// 
+  /// Also see [DecoratedBox].
```

Comment:
We usually put the "See also" in the docs for the class.  Also, we usually explain the relationship between linked symbol and the current symbol.  As an example:

```
See also:

  * [DecoratedBox], which also draws a [Decoration] but is not animated.
```